doc-redirects.properties

```diff
@@ -45,64 +45,13 @@
 
 # Temporary redirect until https://github.com/OpenLiberty/docs/issues/951 is merged
 /docs/latest/why-open-liberty.html=/
-/docs/20.0.0.10/why-open-liberty.html=/
-/docs/20.0.0.11/why-open-liberty.html=/
-/docs/20.0.0.12/why-open-liberty.html=/
-/docs/21.0.0.1/why-open-liberty.html=/
-/docs/21.0.0.2/why-open-liberty.html=/
-/docs/21.0.0.3/why-open-liberty.html=/
-/docs/21.0.0.4/why-open-liberty.html=/
-/docs/21.0.0.5/why-open-liberty.html=/
-/docs/21.0.0.6/why-open-liberty.html=/
-/docs/21.0.0.7/why-open-liberty.html=/
-/docs/21.0.0.8/why-open-liberty.html=/
-/docs/21.0.0.9/why-open-liberty.html=/
-/docs/21.0.0.10/why-open-liberty.html=/
-/docs/21.0.0.11/why-open-liberty.html=/
-/docs/21.0.0.12/why-open-liberty.html=/
-/docs/22.0.0.1/why-open-liberty.html=/
-/docs/22.0.0.2/why-open-liberty.html=/
-/docs/22.0.0.3/why-open-liberty.html=/
-/docs/22.0.0.4/why-open-liberty.html=/
-/docs/22.0.0.5/why-open-liberty.html=/
-/docs/22.0.0.6/why-open-liberty.html=/
-/docs/22.0.0.7/why-open-liberty.html=/
-/docs/22.0.0.8/why-open-liberty.html=/
 
 # Redirect the docs link in the header to the latest version
 /docs/=/docs/latest/
 
 # Redirect 'latest' docs to the latest doc version. Latest is used in the URL by the Doc macros at https://github.com/OpenLiberty/docs/blob/vNext/lib/ol-asciidoc.js. Latest is also used in links to always link to the most recent doc version even when new versions are released.
 /docs/22.0.0.8/*=/docs/latest/
 
-
-/docs/22.0.0.8/=/docs/22.0.0.8/overview.html
-/docs/22.0.0.7/=/docs/22.0.0.7/overview.html
```

Comment:
I do not know the intent for these redirects. I will have to spend more time investigating this redirects.

@steven1046 do you recall why these redirects were added? I found https://github.com/OpenLiberty/docs-playbook/pull/106/files but do not know why the changes are needed.

Reply:
I believe since the work was done to change the URL to display `/latest/` for the latest version this is no longer necessary.
Before this, it was necessary in order to redirect the /latest/ link to the latest numerical version (like /22.0.0.8/).

The only thing that this would change is that someone could no longer type a URL like `/docs/22.0.0.12/` and have it take them to `docs/22.0.0.12/overview.html`.  However, this is only the case for future versions so it would not break any bookmarks so I have no concern.

Reply:
Right, it was just a way to link to the version from places and have it redirect to the overview.html since the slash wasn't working with antora.